install.sh

```diff
@@ -74,19 +160,12 @@ fi
 echo -e "\e[32mDone: Setup Clearpath Robotics package server\e[0m"
 echo ""
 
-
 echo -e "\e[94mUpdating packages and installing ROS 2\e[0m"
 sudo apt -y -qq update
 sudo apt install ros-humble-ros-base python3-argcomplete ros-dev-tools python3-vcstool ros-humble-clearpath-robot python3-clearpath-computer-setup -y
 echo -e "\e[32mDone: Updating packages and installing ROS 2\e[0m"
 echo ""
 
-echo -e "\e[94mSetting up enviroment\e[0m"
```

Comment:
Is there a reason we don't want to still do this, just under the user section?

Reply:
It is below in 288